.github/workflows/test-cli.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: "Test det CLI"
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - 'docs/*'
+  push:
+    branches:
+      - 'main'
+      - 'releases/**'
+
+jobs:
+  test-cli:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9"]
+        os: ["ubuntu-latest", "windows-latest"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install det dependencies
+        run: |
+          python --version
+          python -m pip install --upgrade pip
+          pip --version
+          pip install wheel cryptography patch-ng
+          cd harness; python setup.py bdist_wheel -d ../build
+      - name: Install determined
+        run: |
+          pip install --find-links build determined==0.18.5.dev0
```

Comment:
Working right now on figuring out the source of truth for this value -- if anyone knows, please comment 🙏

Reply:
you mean the version?  We maintain it with bumpversion, see `.bumpversion.cfg` I think.

Reply:
So I should just use the value of `current_version` there?

Reply:
I mean, I think it you configure bumpversion to update this file, then it should be safe to have the literal `0.18.5.dev0` written into the file.

Reply:
Ah, that's an idea as well. While I don't like the idea of potentially adding noise when version bumping, it seems much more reasonable to make the relationship between the files explicit by adding it to that config file. Thanks for the suggestion!

Reply:
(quietly mentions that having a versioned version file is kinda silly when the repo metadata already tracks versions)